KenticoCloud.Delivery.Rx/DeliveryObservableFactory.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+
+using Newtonsoft.Json.Linq;
+
+namespace KenticoCloud.Delivery.Rx
+{
+    public static class DeliveryObservableFactory
```

Comment:
I don't see this class as a factory - I would prepare to have instantiable class with the IDeliveryClient as a parameter in constructor. This would help with injectability and testability when user needs to mock this, let's say wrapper/fasade of a delivery client.